errors.json

```diff
@@ -39,107 +39,96 @@
 			"General"
 		]
 	},
+	"UnsupportedApiVersion": {
+		"description": "The service doesn't support the openEO API version specified in the request URL. Clients should check well-known document for supported versions.",
+		"message": "The requested API version '{version}' is not supported.",
+		"http": 404,
+		"tags": [
+			"General"
+		]
+	},
 	"RequestTimeout": {
 		"description": "The request took too long and timed out.",
 		"message": "Request timed out.",
 		"http": 408,
 		"tags": [
 			"General",
-			"Job Management"
-		]
-	},
-	"CRSInvalid": {
-		"description": "Invalid or unsupported CRS specified.",
-		"message": "CRS '{crs}' is invalid.",
-		"http": 400,
-		"tags": [
-			"Processes"
-		]
-	},
-	"CoordinateOutOfBounds": {
-		"description": null,
-		"message": "Coordinate is out of bounds.",
-		"http": 400,
-		"tags": [
-			"Processes"
+			"Data Processing"
 		]
 	},
 	"BudgetInvalid": {
 		"description": "The budget is too low as it is either smaller than or equal to 0 or below the costs.",
-		"message": "The budget is too low.",
+		"message": "The specified budget is too low.",
 		"http": 400,
 		"tags": [
-			"Job Management",
-			"Secondary Services Management"
+			"Batch Jobs",
+			"Secondary Services",
+			"Data Processing"
 		]
 	},
 	"NoDataForUpdate": {
 		"description": "For PATCH requests: No valid data specified at all.",
-		"message": "No valid data specified to be updated.",
+		"message": "No data specified to be updated.",
 		"http": 400,
 		"tags": [
-			"Job Management",
-			"Secondary Services Management",
-			"Process Graph Management"
+			"Batch Jobs",
+			"Secondary Services"
 		]
 	},
 	"PropertyNotEditable": {
 		"description": "For PATCH requests: The specified parameter can't be updated. It is read-only.",
-		"message": "Property '{property}' is read-only.",
+		"message": "The specified property '{property}' is read-only.",
 		"http": 400,
 		"tags": [
-			"Job Management",
-			"Secondary Services Management",
-			"Process Graph Management"
+			"Batch Jobs",
+			"Secondary Services"
 		]
 	},
 	"CollectionNotFound": {
 		"description": "The requested collection does not exist.",
 		"message": "Collection '{identifier}' does not exist.",
 		"http": 404,
 		"tags": [
-			"EO Data Discovery",
-			"Processes"
+			"EO Data Discovery"
 		]
 	},
 	"StorageFailure": {
 		"description": "Server couldn't store file(s) due to server-side reasons.",
-		"message": "Unable to store file(s).",
+		"message": "Unable to store files due to a server error. Please try again later.",
```

Comment:
> Please try again later

I would not add that. It kind of suggests that a client can just put things in backof-and-retry-loop and it implies some sort of SLA commitment.

Reply:
That's the nature of 500 error codes. It basically says, it's an error on the server side, please try again and this message is for this use case. If it's a client fault, use one of the other "File*" error messages. Or should there be such a general error also with 400 code?

Reply:
I just mean that the "please try again later" will probably shown to the user in some form and it suggests that the problem will go away magically. I think it's better not to imply that, so that the user takes more appropriate action (e.g. contact support) instead just blindly retrying.
But again, this is nitpicking :)

Reply:
Maybe replace "please try again" with "please try again or contact support?"